build.gradle

```diff
@@ -75,12 +75,14 @@ configure(subprojects) {
     }
 
     repositories {
+    	mavenLocal()
         mavenCentral()
         maven { url 'https://jitpack.io' }
     }
 
     dependencies {
         testCompile "junit:junit:$junitVersion"
+        compile 'woodser:monero-wallet-java:0.0.2-SNAPSHOT'
```

Comment:
I think Bisq generally has a "no new jar deps" policy, can you explain why it's really necessary to add a jar dep?

Reply:
I reference same reason prevously:

> This is necessary for the workaround to add Maven project dependencies to Gradlle projects.

The only other way is to create a Gradle project from the **`monero-wallet-java`** Maven project and then maintain this. I rather went with the former solution: compile as a JAR dependency.